website/src/get-started/wordcount-example.md

```diff
@@ -1308,7 +1313,14 @@ static class AddTimestampFn extends DoFn<String, String> {
 ```
 
 ```py
-# This feature is not yet available in the Beam SDK for Python.
+import apache_beam as beam
+from apache_beam.transforms import window
+...
+...
+class AddTimestampFn(beam.DoFn):
+
+  def process(self, element):
+    return window.TimestampedValue(element, self.timestamp_seconds)
```

Comment:
You can match the java snippet by assigning random timestamps in the [min_ts, max_ts) range